server/server.go

```diff
@@ -0,0 +1,64 @@
+package server
+
+import (
+	"context"
+	"net/http"
+
+	microserver "github.com/giantswarm/microkit/server"
+	"github.com/giantswarm/micrologger"
+	kithttp "github.com/go-kit/kit/transport/http"
+
+	"github.com/giantswarm/prometheus-config-controller/service"
+)
+
+type Config struct {
+	Service *service.Service
+
+	MicroServerConfig microserver.Config
+}
+
+func DefaultConfig() Config {
+	return Config{
+		Service: nil,
+
+		MicroServerConfig: microserver.DefaultConfig(),
+	}
+}
+
+func New(config Config) (microserver.Server, error) {
+	newServer := &server{
+		logger: config.MicroServerConfig.Logger,
+
+		config: config.MicroServerConfig,
+	}
+
+	newServer.config.Endpoints = []microserver.Endpoint{}
```

Comment:
Lets please use the usual endpoints like version and healthz here. See https://github.com/giantswarm/kvm-operator/blob/bf49015a93ecd9f793e45ac9297304992521df5e/server/server.go#L79-L81.

Reply:
Yeah, for sure. Next PR